docker/BUILD

```diff
@@ -71,7 +82,7 @@ debug_docker_build(
 )
 
 debug_docker_build(
-    entrypoint = ["/usr/local/bin/pilot"],
+    entrypoint = ["/usr/local/bin/agent"],
```

Comment:
well you can rename the directory to istio-agent (or istio-envoy-agent)
with tab completion long names aren't really a big deal, and we don't expect users to manually run those binaries anyway